src/convertToObject.js

```diff
@@ -6,7 +6,26 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styleObject = {};
+  const declarations = sourceString.split(';');
```

Comment:
The variable name 'declarations' is clear and descriptive, which is good. However, 'styleObject' could be improved to something more descriptive like 'cssStyleObject' to reflect that it represents CSS styles.